crowbar_framework/app/models/api/upgrade.rb

```diff
@@ -24,6 +24,10 @@ def status
       end
 
       def checks
+        upgrade_status = ::Crowbar::UpgradeStatus.new
+        # the check for current_step means to allow running the step at any point in time
+        upgrade_status.start_step if upgrade_status.current_step == "upgrade_prechecks"
```

Comment:
> `upgrade_status.start_step("upgrade_prechecks")`

@aspiers you had some PR for that already, but I cannot find it... 

Other than this (and https://github.com/aspiers/crowbar-core/pull/1) , do you think **this** PR (736) needs to be changed or could we proceed here and adapt the library later?